harvester/harvester.go

```diff
@@ -180,9 +180,16 @@ func (d *Driver) Remove() error {
 		}
 		return err
 	}
+
+	removeAll := false
+	if vm.Annotations["harvesterhci.io/removeAllPersistentVolumeClaims"] == "true" {
```

Comment:
nit: this could be declared as a constant

Reply:
I have also thought about this and think the best place for it would be where [RemovedPVCsAnnotationKey](https://github.com/harvester/harvester/blob/4a35e31b1c487d10e832b6557c4edd42eee768bf/pkg/util/constants.go#L5) is defined which is used a few lines below. But this is in the harvester project; and opening a PR because of one line makes little sense. That's why I decided against it. But I can define it as a constant here in the node driver.